english/marshmallow-taffy/start-conversation/conversation.md

```diff
@@ -2,3 +2,7 @@ You greet Taffy politly, he responds with strange line:
   Welcome back %username%
 
 Do You:
+
+[Write a caffeine-fueled full-length novel?](novel/full-length-novel.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.